server/src/main/resources/org/elasticsearch/bootstrap/security.policy

```diff
@@ -172,4 +172,9 @@ grant {
   permission java.io.FilePermission "/proc/self/cgroup", "read";
   permission java.io.FilePermission "/sys/fs/cgroup/-", "read";
 
+  // needed for com.nimbusds.jose.shaded.gson.Gson.toJson
+  permission java.lang.RuntimePermission "accessDeclaredMembers";
+
+  // needed for com.nimbusds.oauth2.sdk.id.Identifier#value
+  permission java.lang.reflect.ReflectPermission "suppressAccessChecks";
```

Comment:
Can we grant these permissions be granted to the nimbusds codebase, rather than the "global" grant?

Reply:
Duplicate of a different comment. Addressing it in the other comment.